lib/api/web-element/commands/update.js

```diff
@@ -22,10 +22,10 @@
  * @instance
  * @syntax browser.element(selector).update(characters, ...keys)
  * @param {...string} keys
- * @see https://www.w3.org/TR/webdriver#element-send-keys
+ * @link /#element-send-keys
  * @returns {ScopedWebElement}
  */
-module.exports.command = function(...args) {
+module.exports.command = function (...args) {
```

Comment:
I suggest, remove this unnecessary space between as it shows no significant change.